lib/buildS3VersionList.js

```diff
@@ -129,7 +131,9 @@ function getVersion(params) {
             runDeltaOnly = params.runDelta || false;
         }
 
-        s3 = new AWS.S3();
+        s3 = new AWS.S3(
+          {signatureVersion: 'v4'}
```

Comment:
I was under the impression AWS SDK handles this automatically, although I may have missed something - can you explain why we need to specify the signatureVersion to v4?